app/gen-server/migration/1664528376930-UserRefUnique.ts

```diff
@@ -8,13 +7,8 @@ export class UserRefUnique1664528376930 implements MigrationInterface {
     // the ref column unique.
 
     // Update users that don't have unique ref set.
-    const userList = await queryRunner.manager.createQueryBuilder()
```

Comment:
Hi @CamilleLegeron. I have a question, can you try updating this migration, and the one above (`UserUUID`) with a code that tries not to use `User` entity. From my brief tests, it should work.

```
    const userList = await queryRunner.manager.createQueryBuilder()
      .select(["users.id", "users.ref"])
      .from("users", "users")
      .where("ref is null")
      .getMany();
    userList.forEach(u => u.ref = makeId());

```

Here, I'm trying to force typeorm to not product full SELECT, and I'm only specifying columns that I know exists.
If that works, it will simplify this PR a lot.

Reply:
ooh wuaah yes it's works !! So nice and simpler

Reply:
it seems we can not use `save` function with partial entity, but it's possible with `update` function. The weakness it's that we can't use chunk option